Gemfile

```diff
@@ -3,19 +3,19 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.2.2'
+ruby '3.3.0'
```

Comment:
Mainly because of the challenge I thought it would be to update ruby version. It weighted me more on the 'making my environment work' than 'making my app work'. But I guess it was worth. The 3.3.0 xmas version brought performance improvements, YJIT enabled by default (I have to check that later) and the `it` keyword to call self inside a looping block.